LoginWidget.py

```diff
@@ -0,0 +1,272 @@
+        self.NEWUSERRadioButton.setObjectName("NEWUSERRadioButton")
+        self.AVATARFrame = QtWidgets.QFrame(self.main)
+        self.AVATARFrame.setGeometry(QtCore.QRect(190, 250, 61, 61))
+        self.AVATARFrame.setStyleSheet("QFrame {\n"
+                                       "    background-image: url(:/User/img/userv3.png);\n"
+                                       "    border: 7px solid;\n"
+                                       "}\n"
+                                       "")
+        self.AVATARFrame.setFrameShape(QtWidgets.QFrame.StyledPanel)
+        self.AVATARFrame.setFrameShadow(QtWidgets.QFrame.Raised)
+        self.AVATARFrame.setObjectName("AVATARFrame")
+        self.LOGOframe = QtWidgets.QFrame(self.main)
+        self.LOGOframe.setGeometry(QtCore.QRect(130, 10, 201, 171))
+        self.LOGOframe.setStyleSheet("background-image: url(:/LOGO/img/logo.png);")
+        self.LOGOframe.setFrameShape(QtWidgets.QFrame.StyledPanel)
+        self.LOGOframe.setFrameShadow(QtWidgets.QFrame.Raised)
+        self.LOGOframe.setObjectName("LOGOframe")
+        self.NEWUSERLabel = QtWidgets.QLabel(self.main)
+        self.NEWUSERLabel.setGeometry(QtCore.QRect(150, 430, 71, 21))
+        self.NEWUSERLabel.setStyleSheet("color: rgb(200, 200, 200);")
+        self.NEWUSERLabel.setObjectName("NEWUSERLabel")
+        self.horizontalLayout.addWidget(self.main)
+        self.Login.addWidget(self.content)
+        self.bottom = QtWidgets.QFrame(LOGINWidget)
+        self.bottom.setMaximumSize(QtCore.QSize(16777215, 40))
+        self.bottom.setBaseSize(QtCore.QSize(0, 0))
+        self.bottom.setStyleSheet("background-color: rgb(255, 255, 127);")
+        self.bottom.setFrameShape(QtWidgets.QFrame.StyledPanel)
+        self.bottom.setFrameShadow(QtWidgets.QFrame.Raised)
+        self.bottom.setObjectName("bottom")
+        self.Login.addWidget(self.bottom)
+
+        self.retranslateUi(LOGINWidget)
+        QtCore.QMetaObject.connectSlotsByName(LOGINWidget)
+        self.LOGINButton.clicked.connect(self.login)
+        self.ERRORXButton.clicked.connect(self.closePopup)
+        self.ERRORFrame.hide()
+
+    def retranslateUi(self, LOGINWidget):
+        _translate = QtCore.QCoreApplication.translate
+        LOGINWidget.setWindowTitle(_translate("LOGINWidget", "LOGIN"))
+        self.ERRORLabel.setText(_translate("LOGINWidget", "ERROR"))
+        self.ERRORXButton.setText(_translate("LOGINWidget", "X"))
+        self.USERLabel.setPlaceholderText(_translate("LOGINWidget", "USER"))
+        self.PASSWORDLabel.setPlaceholderText(_translate("LOGINWidget", "PASSWORD"))
+        self.LOGINButton.setText(_translate("LOGINWidget", "LOGIN "))
+        self.NEWUSERLabel.setText(_translate("LOGINWidget", "NEW USER"))
+
+
+    def __init__(self, home):
+        self.home = home
+
+    def login(self):
+        db = TicketAir.DataBase()
+        if self.NEWUSERRadioButton.isChecked():
+            if not db.nickTaken(self.USERLabel.text()):
```

Comment:
Make a nick a key in users table. This operation is not atomic and assuming there is possibility for two users to be created at the same time this is a problem.